lib/client.js

```diff
@@ -18,15 +18,111 @@ var uuid = require('node-uuid').v4;
 
 require('./config');
 
+var api = {};
+module.exports = api;
+
+var logger = bedrock.loggers.get('app');
+
 // configure for tests
 bedrock.events.on('bedrock.test.configure', function() {
   require('./test.config');
 });
 
-var api = {};
-module.exports = api;
+bedrock.events.on('bedrock-mongodb.ready', function(callback) {
+  if(!config['messages-client'].enable) {
+    return callback(); 
+  }
+  var interval = 1;
+  config['messages-client'].clients
+    .forEach(function(clientOptions) {
+      var jobId = createJobId(clientOptions.endpoint);
+      var endpoint = clientOptions.endpoint;
+      var privateKeyPem = clientOptions.privateKeyPem;
+      var publicKeyId = clientOptions.publicKeyId;
+      var strictSSL = !('strictSSL' in clientOptions) ?
+        true : !!clientOptions.strictSSL;
+      var fn = function(job, callback) {
+        var options = {
+          job: job,
+          endpoint: endpoint,
+          privateKeyPem: privateKeyPem,
+          publicKeyId: publicKeyId,
+          strictSSL: strictSSL
+        };
+        pollServer(options, function(err, results) {
+          if(err) {
+            callback(err);
+          }
+          callback(null);
+        });
+      };
 
-var logger = bedrock.loggers.get('app');
+      scheduler.define(jobId, fn);
+
+      api.start(endpoint, interval, function(err) {
+        if(err) {
+          callback(err);
+        }
+      });
+    });
+});
+
+/*
+ * Starts a client that polls the endpoint with the specified interval setting.
+ * If a client is already running on the specified endpoint, it will be
+ * restarted with the new time interval.
+ *
+ * @param endpoint The endpoint for the client to poll
+ * @param interval 1, 5, or 10 minute intervals to poll the endpoint
```

Comment:
Change to `@param interval the number of minutes to wait between polling the endpoint (e.g. 1, 5, or 10).`